app/api/views/views.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask, jsonify, abort, make_response, Blueprint
+from flask_restful import Api, Resource, reqparse
+from app.api.models.request import Requests
+app_bp = Blueprint('app', __name__)
```

Comment:
add a new line here between line 3 and line 4. consider using a linter `pylint` or `flake8`. They point these errors out. If you are using `PyCharm`, right click and select format document